OpenCL_C.txt

```diff
@@ -6303,6 +6303,11 @@ pointed to by obj to the value value.
 
 [source,c]
 ----------
+// Requires OpenCL C 2.0 or newer.
```

Comment:
Should this be "OpenCL C **2.0** or newer" or "OpenCL C **3.0** or newer"?

I'm not sure if it's observable, but the function overloads for the named (non-generic) address spaces were not in previous OpenCL C specifications.

This same comment applies elsewhere in this PR, also.

Reply:
Yes, you're correct.  It should say 3.0.  I'll fix that.

Reply:
Fixed.